test/controllers/works_controller_test.rb

```diff
@@ -0,0 +1,153 @@
+require "test_helper"
+
+describe WorksController do
+  describe "index" do
+    it "will return success status" do
+      get works_path
+      must_respond_with :success
+    end # it "will return success status" do
+  end # index
+
+  describe "root" do
+    # check that the index page works
+    it "returns success status" do
+      # act
+      get root_path
+      # assert
+      must_respond_with :success
+    end
+  end # index
+
+  describe "new" do
+    it "will return a success status" do
+      get new_work_path
+      must_respond_with :success
+    end #it "will return a success status" do
+  end # describe "new" do
+
+  describe "create" do
+    it "adds the work to the DB and then redirects to the works_path when work data is valid" do
+      # Arrange
+      work_data = {
+        work: {
+          category: "book",
+          title: "HP 3",
+          creator: "J.K Rowling"
+        }
+      }
+      new_work = Work.new(work_data[:work]).must_be :valid?
+
+      start_count_work = Work.count
+
+      # Act
+      post works_path, params: work_data
+
+      # Assert
+      must_respond_with :redirect
+      must_redirect_to works_path
+
+      Work.count.must_equal start_count_work + 1
+    end # "it adds the work..."
+
+    it "returns bad_request when the work data is invalid" do
+      # QUESTION: what is invalid data? add this test if we add validations to the Work model
+    end # it "returns bad_request when the work data is invalid" do
```

Comment:
Here bad data would be anything that makes the `Work` validations fail, for example not having a title. The goal is not to test your specific validations, but to test that this action can handle bad data in general.